astropy/io/fits/hdu/table.py

```diff
@@ -702,6 +686,17 @@ class TableHDU(_TableBaseHDU):
         r'(?P<code>[ADEFIJ])(?P<width>\d+)(?:\.(?P<prec>\d+))?')
 
     def __init__(self, data=None, header=None, name=None):
+        """
+        Parameters
+        ----------
+        header : `Header`
+            header to be used
```

Comment:
It did not matter before because the docstring was in a hidden class, but now that it is not anymore, the presentation needs to be cleaner:

* Move this docstring to under `class TableHDU`, as done everywhere else in Astropy. There is no need for docstring in `__init__`.
* Capitalize first word in description.
* Add a period at the end of the description.
* Also would be easier on the eyes if the order of parameters here reflect the order in the method definition above.